keras/callbacks.py

```diff
@@ -605,6 +616,31 @@ def __init__(self, log_dir='./logs',
         self.merged = None
         self.write_graph = write_graph
         self.write_images = write_images
+        self.embeddings_freq = embeddings_freq
+        self.embeddings_layer_names = embeddings_layer_names
```

Comment:
by default you could maybe add all embedding layers' names by checking the type of each layer in the model so that it doesn't affect the naming when building the model. You could then bypass the creation of a `tf.train.Saver` by checking if `embeddings_freq` is not 0.